.github/renovate.json5

```diff
@@ -5,7 +5,9 @@
     // Each ignore is probably connected with an ignore in pyproject.toml.
```

Comment:
Per this comment, did you need to also add an ignore in pyproject.toml?

Reply:
In this case no, because we don't even have tox-gh in pyproject.toml